### ISSUE TITLE
Tabs break when there is only one tab in <Tabs.List>

### ISSUE THREAD

**Issue by joeyamst** [Amsterdam/design-system#1774]
# Describe the issue

When using the Tabs, it breaks with the error `Uncaught TypeError: children[0].props.children.map is not a function` in the `<Tabs>` component.

## Summary

```
            <Tabs activeTab={0}>
                <Tabs.List>
                    <Tabs.Button tab={0}>Tab 1</Tabs.Button>
                    {/*<Tabs.Button tab={1}>Tab 2</Tabs.Button>*/}
                </Tabs.List>
                <Tabs.Panel tab={0}>
                    <Heading level={3}>Tab 1 Title</Heading>
                </Tabs.Panel>
            </Tabs>
```
This breaks with the error. Uncommenting line 4 and it works fine again and shows 2 tabs.

## Environment

- Browser: Chrome
- Operating System: Windows with WSL
- Version of the design system: v0.13.1

## Checklist

Before submitting your issue, please ensure you have done the following:

- [ x ] Search the issue tracker to ensure the issue has not already been reported.
- [ x ] Verify that the issue can be consistently reproduced.
- [ x ] Check if the issue is already documented.

## Additional notes

I agree having only 1 tab is useless, but for developing sometimes you build one tab first, and add more later. Maybe tabs can also be dynamic on user input, creating a situation where it's possible to only having one tab.

**Comment by RubenSibon** [comment]
A fix for this is incoming.

**Comment by VincentSmedinga** [comment]
This has been resolved but has yet to be published. We expect a new release to ship in the second half of January. If it would be useful to have it earlier, let us know, and we can make it a hotfix.

**Comment by VincentSmedinga** [comment]
@joeyamst The fix has been published in release 0.14.
